src/test/java/my/test/example/MainTest.java

```diff
@@ -0,0 +1,31 @@
+package my.test.example;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+
+public class MainTest {
+    LoginPage loginPage;
+
+    @BeforeEach
+    void setup() {
+        loginPage = new LoginPage();
+
+    }
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/EmailCredentials.csv", numLinesToSkip = 1)
+    void loginTest(String login, String password) {
+        loginPage.login(login, password);
+        Assertions.assertEquals(loginPage.emailPageIsPresent(), true);
+        loginPage.logOutCurrentUser();
```

Comment:
2 points:
1. Do not put anything after assertion, because this method will be not executed in case if assertion fails. Use "after" hooks for such purposes
2. I'm not sure that you need to logout for this test at all